crates/symbolicator/src/services/symbolication/symbolication.rs

```diff
@@ -272,6 +322,8 @@ fn symbolicate_frame(
                 package: lookup_result.object_info.raw.code_file.clone(),
                 addr_mode: lookup_result.preferred_addr_mode(),
                 instruction_addr,
+                il_offset: None,
+                function_index: frame.function_index,
```

Comment:
we copy the `function_index`, but set the `il_offset` to `None`. maybe we should also just copy that over.